libavfilter/dnn/dnn_backend_common.h

```diff
@@ -26,6 +26,42 @@
 
 #include "../dnn_interface.h"
 
+// one task for one function call from dnn interface
+typedef struct TaskItem {
+    void *model;
+    AVFrame *in_frame;
+    AVFrame *out_frame;
+    const char *input_name;
+    const char **output_names;
+    int async;
+    int do_ioproc;
```

Comment:
Since these have value 0 or 1, will `uint8_t` be better?

Reply:
yes